content/ecosystem/bridges/discord/bridges.toml

```diff
@@ -114,5 +114,5 @@ threads = true
 redactions = true
 editing = true
 reactions = true
-presence = false
-typing_notifications = false
+presence = true
+typing_notifications = true
```

Comment:
Hey @cloudrac3r, I am generally willing to accept your proposed change.

I can find traces of presence being supported in both directions:
https://gitdab.com/cadence/out-of-your-element/src/commit/60a53f76bb001fe924731ca84dabe51e54127d16/readme.md?display=source#L23

I can find this trace of typing being supported from discourse to matrix
https://gitdab.com/cadence/out-of-your-element/src/commit/60a53f76bb001fe924731ca84dabe51e54127d16/src/d2m/event-dispatcher.js#L330

, but nothing for matrix to discourse. Can you point me at something for typing notifications m2d?

Reply:
m->d typing notifications are waiting on synapse to support the stable endpoints before they can be implemented. see https://github.com/element-hq/synapse/pull/17881

Reply:
I see, thank you. Please do let us know when this lands so we can update the support properly. :)